app/DoctrineMigrations/Version208.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Adding management costs table
+ */
+class Version208 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() != 'postgresql', 'Migration can only be executed safely on \'postgresql\'.');
+
+        $this->addSql('CREATE TABLE prof_deputy_management_cost (id SERIAL NOT NULL, amount DOUBLE PRECISION DEFAULT NULL, prof_deputy_management_cost_type_id VARCHAR(255) NOT NULL, has_more_details BOOLEAN NOT NULL, more_details TEXT DEFAULT NULL, PRIMARY KEY(id))');
+    }
+
+    /**
+     * @param Schema $schema
+     */
+    public function down(Schema $schema)
+    {
+        // this down() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() != 'postgresql', 'Migration can only be executed safely on \'postgresql\'.');
+
+        $this->addSql('CREATE SCHEMA public');
+        $this->addSql('DROP TABLE prof_deputy_management_cost');
+    }
+}
```

Comment:
this needs to be just a new field as part of report. Add the property to the estimates trait and it should just alter report table to add the new field

Reply:
Ahhh OK, will update!